index.d.ts

```diff
@@ -3,6 +3,7 @@ import { Options as SnakeCaseOptions } from "snake-case";
 
 // eslint-disable-next-line @typescript-eslint/ban-types
 type EmptyTuple = [];
+type ObjectOptional = Record<string, unknown> | undefined;
```

Comment:
Type definitions for optional properties such as DeepObjectType defined in the type test